LICENSE.txt

```diff
@@ -25,22 +25,39 @@ jetty Web Server Library (As-Is, No Modifications)
   http://eclipse.org/jetty/
 
 
+Apache log4j (As-Is, No Modifications)
+  Copyright (C) 1999-2005 The Apache Software Foundation
+  Apache License, Version 2.0 (attached), with Copyright Notice
+  https://logging.apache.org/
+
+
 Apache PDFBox (As-Is, No Modifications)
   Copyright (C) 2009–2015 The Apache Software Foundation
   Apache License, Version 2.0 (attached), with Copyright Notice
   https://pdfbox.apache.org/
 
 
 jSSC Library (As-Is, No Modifications)
-  Copyright (c) 2010-2013 Alexey Sokolov (scream3r) 
```

Comment:
This whitespace change is OK, but let's be careful when we're using stuff like Atom text editor, it'll do whitespace cleanup all over the files that we may not have asked for. :)

Reply:
Noted 🤦‍♂️